packages/extension-polkagate/src/popup/notification/util.ts

```
@@ -0,0 +1,547 @@
+export interface StakingRewardInformation {
+  address: string;
+  data: PayoutsProp[];
+  network: DropdownOption;
+}
+
+export function timestampToDate (timestamp: number | string, format: 'full' | 'short' | 'relative' = 'full'): string {
+  // Ensure timestamp is a number and convert if it's a string
+  const timestampNum = Number(timestamp);
+
+  // Check if timestamp is valid
+  if (isNaN(timestampNum)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)